.hlint.yaml

```diff
@@ -86,7 +86,7 @@
   - {name: ViewPatterns, within: []}
 
   # Shady extensions
-  - {name: CPP, within: [DA.Sdk.Cli.System, DA.Sdk.Cli.Version]}
+  - {name: CPP, within: [DA.Sdk.Cli.System, DA.Sdk.Cli.Version, DAML.Assistant.Install]}
```

Comment:
Could you move the part that requires CPP / conditional comiplation to a separate module `DAML.Assistant.Install.Win32`?